extensions/community_extensions/dsa_transparency.md

```diff
@@ -0,0 +1,345 @@
+<p>This technical specification provides data formats and transport for the advertising industry to implement relevant DSA transparency information. This solution should be applicable across most relevant use cases including; programmatic and non-programmatic media buys, channels including desktop web, mobile (web/app), video, CTV.</p>
+<h2>Object Specification for OpenRTB 2.X</h2>
+<h3>Bid Request</h3>
+<h4>Object: Regs</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>ext.dsa</code></td>
+<td>object&nbsp;</td>
+<td>Extension for DSA transparency information</td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Object: DSA</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>dsainfo</code></td>
+<td>integer</td>
+<td>Flag to indicate if DSA information should be made available. This will signal if the bid request belongs to an Online Platform/VLOP, such that a buyer should respond with DSA Transparency information based on the pubrender value.
+    <p><code>0</code> = Not required</p>
+    <p><code>1</code> = Supported, bid responses with or without DSA object will be accepted</p>
+    <p><code>2</code> = Required, bid responses without DSA object will not be accepted</p>
+    <p><code>3</code> = Required, bid responses without DSA object will not be accepted, Publisher is an Online Platform</p></td>
+</tr>
+<tr>
+<td><code>pubrender</code></td>
+<td>integer</td>
+<td>Flag to indicate if the publisher will render the DSA Transparency info. This will signal if the publisher is able to and intends to render the icon and display the DSA transparency info to the end user.
+    <p><code>0</code> = Publisher can't render</p>
+    <p><code>1</code> = Publisher could render depending on adrender</p>
+    <p><code>2</code> = Publisher will render</p></td>
+</tr>
+<tr>
+<td><code>datatopub</code></td>
+<td>integer</td>
+<td>Independent of pubrender, the publisher may need the transparency data for audit purposes.
+    <p><code>0</code> = do not send transparency data</p>
+    <p><code>1</code> = optional to send transparency data</p>
+    <p><code>2</code> = send transparency data</p></td>
+</tr>
+<tr>
+<td><code>transparency</code></td>
+<td>array of object</td>
+<td>Array of objects of the entities that applied user parameters and the parameters they applied.</td>
```

Comment:
Could these entities and parameters they applied differ on a per-impression (ad slot) basis in terms of the sellside targeting? 
If so, should this transparency information be part of the `Imp` object?

Reply:
The GPWG discussed and would like to tackle this post-MVP which this extension is meant to be.

Reply:
Please start values with 1 and leave 0 for undknown/not set